tests/foreman/cli/test_activationkey.py

```diff
@@ -1731,3 +1731,65 @@ def test_positive_ak_with_custom_product_on_rhel6(module_org, rhel6_contenthost,
     rhel6_contenthost.install_katello_ca(target_sat)
     result = rhel6_contenthost.register_contenthost(module_org.label, activation_key=ak.name)
     assert 'The system has been registered with ID' in result.stdout
+
+
+@pytest.mark.tier3
+def test_positive_invalid_release_version(module_entitlement_manifest_org, module_target_sat):
```

Comment:
I doubt  `module_entitlement_manifest_org` would work in 6.15 and above, as this fixture retuns SCA disable orgnization.
I would suggest to use `module_sca_manifest_org` for 6.15 and above